README.md

```diff
@@ -1,2 +1,19 @@
-# media
-Drupal 8 version of the Drupal media module.
+# Media Module
+
+A complete media module for Drupal 8.
+
+## Requirements
+
+* Latest dev release of Drupal 8.x.
+
+## Configuration
+
+@ToDo
+
+## Technical details
+
+@ToDo
+
+## Maintainers
+
+@ToDo
```

Comment:
Also take a look at: https://www.drupal.org/node/2181737 for the recommended readme file structure. You're missing a couple of sections.